crates/autopilot/src/domain/settlement/trade/math.rs

```diff
@@ -441,6 +410,27 @@ impl Trade {
     }
 }
 
+/// Fee per trade in a solution. These fees are taken for the execution of the
+/// trade.
+#[derive(Debug, Clone)]
+pub struct FeeBreakdown {
+    /// Fee reported by solvers, representing difference between UCP and custom
+    /// prices.
+    pub executed: eth::TokenAmount,
+    /// Breakdown of protocol fees.
+    pub protocol: Vec<ExecutedProtocolFee>,
```

Comment:
The order of items also indicates the order in which the fee policies got applied, right?
Seems important enough to point out in a comment.